ihatemoney/static/js/ihatemoney.js

```diff
@@ -5,3 +5,46 @@ function selectCheckboxes(value){
     els[i].checked = value;
   }
 }
+
+function updateCheckBoxesFromPrivacySelect() {
+  var history_checkbox = document.getElementById('logging_enabled');
+  var record_ip_checkbox = document.getElementById('record_ip');
+  var record_ip_checkbox_text = document.getElementById("record_ip_label");
+  var select_input = document.getElementById("logging_preferences");
+
+  if (select_input.selectedIndex === 0) {
+    history_checkbox.checked = false;
+    record_ip_checkbox.checked = false;
+    record_ip_checkbox.disabled = true;
+    record_ip_checkbox_text.classList.add("text-muted");
+  } else if (select_input.selectedIndex === 1 || select_input.selectedIndex === 2) {
+    history_checkbox.checked = true;
+    record_ip_checkbox.disabled = false;
+    record_ip_checkbox_text.classList.remove("text-muted");
+    if (select_input.selectedIndex === 2) {
+      record_ip_checkbox.checked = true
+    }
+  }
+}
+
+function updatePrivacySelectFromCheckBoxes() {
+  var history_checkbox = document.getElementById('logging_enabled');
+  var record_ip_checkbox = document.getElementById('record_ip');
+  var record_ip_checkbox_text = document.getElementById("record_ip_label");
+  var select_input = document.getElementById("logging_preferences");
+
+  if (!history_checkbox.checked) {
+    record_ip_checkbox.checked = false;
+    record_ip_checkbox.disabled = true;
+    record_ip_checkbox_text.classList.add("text-muted");
+    select_input.selectedIndex = 0
+  } else {
+    record_ip_checkbox.disabled = false;
+    record_ip_checkbox_text.classList.remove("text-muted");
+    if (record_ip_checkbox.checked){
+      select_input.selectedIndex = 2
+    } else {
+      select_input.selectedIndex = 1
+    }
+  }
```

Comment:
If I understand properly, this javascript is here to grey out the checkbox if we deselect the project history. I think we can simplify that a lot, and I believe having two checkboxes which are not correlated together might work just okay.

Something like :
[ ] Enable project history
[ ] Use IP tracking for project history

Without JS at all :-)

Reply:
Done as of 0b206da